### DISCUSSION TITLE
Detect Angular project in workspace

### DISCUSSION THREAD

**Discussion by RiccardoViola** [lucono/karma-test-explorer#64] [Issues / Help] [answered]
Hi, 
I read the documentation but I didn't understand how to enable the extension to find the Angular projects inside the workspace. I saw that probably this is possible using the karmaTestExplorer.projectWorkspaces configuration specifying the relative paths, but I don't really understand how it works. Maybe it will be clear with an example.
Thank you

**Comment by lucono** [comment, marked as answer]
The [extension settings](https://github.com/lucono/karma-test-explorer/blob/master/docs/documentation.md#extension-settings) section describes how to use that setting in some detail (the string and object formats), and VS Code will also provide auto-completion, descriptions, and validation for it when you're actually adding the setting to your config file because Karma Test Explorer includes a detailed schema for that setting. But I do agree examples would help, so I've also added [some](https://github.com/lucono/karma-test-explorer/blob/master/docs/documentation.md#configuration-examples-using-the-projectworkspaces-option) to the docs in the just released [`v0.8.1`](https://github.com/lucono/karma-test-explorer/blob/master/CHANGELOG.md#081---jan-30-2023) version.

In general, Karma Test Explorer should automatically detect your Angular workspace in scenarios where it is at the root of the VS Code workspace that's open in the IDE. In any scenario where it doesn't automatically detect it, or where you're working with either a multi-root VS Code workspace or a monorepo having the project (one or more) at some path below the root, you should be able to use the `karmaTestExplorer.projectWorkspaces` setting to specify and/or configure the projects.

**Reply by RiccardoViola** [reply]
I had also to "restart" the extension to get the new settings work, but now it's perfect.

Thank you!
